test/compile-test.cpp

```diff
@@ -0,0 +1,51 @@
+#include <cstdio>
```

Comment:
Let's worry about tests next iteration, where we'll specify such things in the CMake file.